drivers/spi/spi_mcux_dspi.c

```diff
@@ -143,35 +577,44 @@ static int spi_mcux_configure(const struct device *dev,
 
 	DSPI_MasterGetDefaultConfig(&master_config);
 
+	master_config.whichPcs = config->whichPcs;
+	master_config.whichCtar = config->whichCtar;
+	master_config.pcsActiveHighOrLow =
+		(spi_cfg->operation & SPI_CS_ACTIVE_HIGH) ?
+			kDSPI_PcsActiveHigh :
+			kDSPI_PcsActiveLow;
+	master_config.samplePoint = config->samplePoint;
+	master_config.enableContinuousSCK = config->enableContinuousSCK;
+	master_config.enableRxFifoOverWrite = config->enableRxFifoOverWrite;
+	master_config.enableModifiedTimingFormat =
+		config->enableModifiedTimingFormat;
+
 	if (spi_cfg->slave > FSL_FEATURE_DSPI_CHIP_SELECT_COUNT) {
-		LOG_ERR("Slave %d is greater than %d",
-			    spi_cfg->slave, FSL_FEATURE_DSPI_CHIP_SELECT_COUNT);
+		LOG_ERR("Slave %d is greater than %d", spi_cfg->slave,
+			FSL_FEATURE_DSPI_CHIP_SELECT_COUNT);
 		return -EINVAL;
 	}
 
 	word_size = SPI_WORD_SIZE_GET(spi_cfg->operation);
 	if (word_size > FSL_FEATURE_DSPI_MAX_DATA_WIDTH) {
-		LOG_ERR("Word size %d is greater than %d",
-			    word_size, FSL_FEATURE_DSPI_MAX_DATA_WIDTH);
+		LOG_ERR("Word size %d is greater than %d", word_size,
+			FSL_FEATURE_DSPI_MAX_DATA_WIDTH);
 		return -EINVAL;
 	}
 
 	ctar_config->bitsPerFrame = word_size;
 
-	ctar_config->cpol =
-		(SPI_MODE_GET(spi_cfg->operation) & SPI_MODE_CPOL)
-		? kDSPI_ClockPolarityActiveLow
-		: kDSPI_ClockPolarityActiveHigh;
+	ctar_config->cpol = (SPI_MODE_GET(spi_cfg->operation) & SPI_MODE_CPOL) ?
+				    kDSPI_ClockPolarityActiveLow :
+				    kDSPI_ClockPolarityActiveHigh;
```

Comment:
Please move all the whitespace changes to a separate commit

Reply:
ok, revert changes